lib/rules/declaration-property-value-no-unknown/index.js

```diff
@@ -28,6 +28,8 @@ const meta = {
 	url: 'https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown',
 };
 
+const CONDITIONAL_RULES_ALLOWED_IN_NESTING = /^(?:media|supports|layer|scope|container)$/i;
```

Comment:
[ask] This isn't required, but does defining these rules in `reference/atKeywords.js` make sense like this?

```js
// https://drafts.csswg.org/css-nesting/#conditionals
const nestingSupportedAtKeywords = new Set([
	'container',
	'layer',
	'media',
	'scope',
	'supports',
]);
```

This might be available for other rules in the future. Any thoughts?

Reply:
> Any thoughts?

It might be useful for #6421
see https://www.w3.org/TR/css-nesting-1/#example-f2b8b339